requirements.txt

```diff
@@ -1 +1,4 @@
 genologics
+couchdb
+numpy
+click
```

Comment:
Why numpy, why couchdb, and frankly, why click since an epp is never triggered manually other than in the testing period ?